main.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+
+def get_list_of_users_names() -> list:
```

Comment:
Function name could be more specific based on pep8 standards.  i.e. including that it's a user input function.